src/m365/spo/commands/site/site-recyclebinitem-restore.spec.ts

```diff
@@ -117,7 +138,63 @@ describe(commands.SITE_RECYCLEBINITEM_RESTORE, () => {
     assert.equal(result, undefined);
   });
 
-  it('catches error when restores all items from recyclebin', async () => {
+  it('restores all items from the first-stage recycle bin', async () => {
+    sinon.stub(request, 'post').callsFake(async (opts) => {
+      if (opts.url === 'https://contoso.sharepoint.com/_api/web/RecycleBin/RestoreAll') {
+        return;
+      }
+
+      throw 'Invalid request';
+    });
+
+    await command.action(logger, {
+      options: {
+        siteUrl: 'https://contoso.sharepoint.com',
+        allPrimary: true
+      }
+    });
+  });
```

Comment:
Currently we can just remove the POST request from the command and this test will still succeed.
Let's add an assert that the post stub is called once.